mod-source-record-storage-server/src/main/java/org/folio/rest/impl/ModTenantAPI.java

```diff
@@ -58,22 +65,31 @@ public void postTenant(TenantAttributes entity, Map<String, String> headers, Han
       if (ar.failed()) {
         handlers.handle(ar);
       } else {
-        setLoadSampleParameter(entity, context)
-          .compose(v -> createStubSnapshot(context, entity))
-          .compose(v -> createStubData(entity))
-          .setHandler(event -> handlers.handle(ar));
+        registerModuleToPubsub(entity, headers, context.owner())
+          .setHandler(registrationAr -> {
+            if (registrationAr.failed()) {
+              handlers.handle(Future.failedFuture(registrationAr.cause()));
```

Comment:
I don't think we should fail tenant registration because of pubsub registration failure at this point. Just for now, let's not disrupt data-import flow when we transitioning towards pubsub.